From: John Garry <john.garry@huawei.com>
Date: Fri, 25 Jan 2019 22:22:39 +0800
Subject: scsi: hisi_sas: Add missing seq_printf() call in
 hisi_sas_show_row_32()
Patch-mainline: v5.1-rc1
Git-commit: ede2afb9c8ed784cdf1b6de9cd02f044668b842a
References: bsc#1140683

This call must have been missed when I reworked the debugfs feature for
upstreaming, so add it back.

Signed-off-by: John Garry <john.garry@huawei.com>
Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
Acked-by: Daniel Wagner <dwagner@suse.de>
---
 drivers/scsi/hisi_sas/hisi_sas_main.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/scsi/hisi_sas/hisi_sas_main.c
+++ b/drivers/scsi/hisi_sas/hisi_sas_main.c
@@ -2692,6 +2692,7 @@ static int hisi_sas_show_row_32(struct s
 	/* completion header size not fixed per HW version */
 	seq_printf(s, "index %04d:\n\t", index);
 	for (i = 1; i <= sz / 4; i++, ptr++) {
+		seq_printf(s, " 0x%08x", le32_to_cpu(*ptr));
 		if (!(i % 4))
 			seq_puts(s, "\n\t");
 	}
